rocketpool/node/collectors/node-collector.go

```diff
@@ -445,9 +459,34 @@ func (collector *NodeCollector) Collect(channel chan<- prometheus.Metric) {
 	if totalRplAtNextCheckpoint < 0 {
 		totalRplAtNextCheckpoint = 0
 	}
+
+	/*
+	 * Calculates a RPIP-30 RPL reward estimate. Assumes that RPIP-30 has been fully phased in
+	 *
+	 * Formula:
+	 * 		current_node_weight / (current_node_weight + previous_interval_total_node_weight) * estimated_collateral_rewards
+	 *
+	 * This will inevitably be inaccurate due to factors external to the node, such as large nodes running
+	 * several bond reductions, or really anything that can move the needle on total_node_weight. However,
+	 * short of generating a full rewards tree at a regular interval to populate this metric, this is the
+	 * best estimator available.
+	 * RPIP-30 is still being phased in, and this estimator will be more and more accurate as it is, but never fully accurate
+	 */
 	estimatedRewards := float64(0)
 	if totalEffectiveStake.Cmp(big.NewInt(0)) == 1 {
-		estimatedRewards = rewardableStakeFloat / eth.WeiToEth(totalEffectiveStake) * totalRplAtNextCheckpoint * nodeOperatorRewardsPercent
+		// (current_node_weight + previous_interval_total_node_weight)
+		nodeWeightSum := big.NewInt(0).Add(nodeWeight, previousIntervalTotalNodeWeight)
+
+		// Convert to float so we can divide them safely
+		nodeWeightFloat := big.NewFloat(0).SetInt(nodeWeight)
+		nodeWeightSumFloat := big.NewFloat(0).SetInt(nodeWeightSum)
```

Comment:
don't bother with big floats. Accuracy loss is meaningless compared to the heuristic's natural inaccuracy.

Call `nodeWeight.Float64()` as early as is convenient and use native floats thereafter.

Reply:
Done

Reply:
As discussed offline, we ran in to some problems here. The version of golang that the docker build uses (1.19.5) does not have a big.Int.Float64() function (See: https://pkg.go.dev/math/big@go1.19.5#Int). Therefore, I'd like to propose keeping the big.Float Quo() division so that the result is unambiguously correct.